docs/registry/upload/_index.md

```diff
@@ -317,18 +334,105 @@ jobs:
 The `build-action` GitHub action relies on a build command that you need to specify in the <file>meta.json</file> file that you created for your module when you first [uploaded it](/registry/upload/#upload-a-custom-module-using-the-cli).
 At the end of your <file>meta.json</file>, add the build configuration:
 
+{{< tabs >}}
+{{% tab name="Single Build File" %}}
+
 ```json {class="line-numbers linkable-line-numbers" data-line="4-7"}
 {
   "module_id": "example-module",
   ...
   "build": {
-    "build": "make module.tar.gz",
-    "arch" : ["linux/amd64", "linux/arm64", "darwin/arm64"]
+    "build": "sh build.sh",          // command that will build your module
```

Comment:
as above, would do `./build.sh` to fix sh / bash issue

Reply:
Thank you! yeah sorry I didn't implement earlier, was slightly confused about if I had to instruct them to make it executable or do something else differently this way